backend/audit/fixtures/workbooks/sac_creation.py

```diff
@@ -0,0 +1,372 @@
+    audit.validators.validate_general_information_json(general_information)
+
+    return general_information
+
+
+# TODO: Pull this from actual information.
+def _fake_audit_information(dbkey, auditee_name=None):
+    gobj: Gen = Gen.select().where(Gen.dbkey == dbkey).first()
+    cfdas = Cfda.select().where(Cfda.dbkey == dbkey)
+
+    agencies = {}
+    cfda: Cfda
+    for cfda in cfdas:
+        agencies[int((cfda.cfda).split(".")[0])] = 1
+
+    findings = Finding.select().where(Finding.dbkey == dbkey)
+    finding: Finding
+    gaap_results = {}
+    # THIS IS NOT A GOOD WAY TO DO THIS, BUT IT IS CLOSE.
+    # IT IS FOR TEST DATA...
+    for finding in findings:
+        if finding.modifiedopinion == "Y":
+            gaap_results["unmodified_opinion"] = 1
+        if finding.materialweakness == "Y":
+            gaap_results["adverse_opinion"] = 1
+        if finding.significantdeficiency == "Y":
+            gaap_results["disclaimer_of_opinion"] = 1
+
+    audit_information = {
+        "agencies": list(
+            map(lambda i: str(i) if len(str(i)) > 1 else f"0{str(i)}", agencies.keys())
+        ),
+        "dollar_threshold": 750000,
+        "gaap_results": list(gaap_results.keys()),
+        "is_aicpa_audit_guide_included": True
+        if gobj.reportablecondition == "Y"
+        else False,
+        "is_going_concern_included": True if gobj.goingconcern == "Y" else False,
+        "is_internal_control_deficiency_disclosed": True
+        if gobj.materialweakness == "Y"
+        else False,
+        "is_internal_control_material_weakness_disclosed": True
+        if gobj.materialweakness_mp == "Y"
+        else False,
+        "is_low_risk_auditee": False,
+        "is_material_noncompliance_disclosed": True
+        if gobj.materialnoncompliance == "Y"
+        else False,
+    }
+
+    audit.validators.validate_audit_information_json(audit_information)
+
+    return audit_information
+
+
+def _create_test_sac(user, auditee_name, dbkey):
+    """Create a single example SAC."""
+    SingleAuditChecklist = apps.get_model("audit.SingleAuditChecklist")
+
+    try:
+        exists = SingleAuditChecklist.objects.get(
+            report_id=dbkey_to_test_report_id(Gen, dbkey)
+        )
+    except SingleAuditChecklist.DoesNotExist:
+        exists = None
+    if exists:
+        exists.delete()
+
+    sac = SingleAuditChecklist.objects.create(
+        submitted_by=user,
+        general_information=_fake_general_information(dbkey, auditee_name),
+        audit_information=_fake_audit_information(dbkey, auditee_name),
+    )
+    # Set a TEST report id for this data
+    sac.report_id = dbkey_to_test_report_id(Gen, dbkey)
+
+    Access = apps.get_model("audit.Access")
+    Access.objects.create(
+        sac=sac,
+        user=user,
+        email=user.email,
+        role="editor",
+    )
+    # Why not give me all the access? This way, I can run the test all the way through!
```

Comment:
Question for later: when we get to cross-validation, using the same access credentials for certifying 'tee and 'tor should fail, right?